docs/reference/comparison-to-scala.md

```diff
@@ -32,8 +32,14 @@ Taking this into account, if you are happy with Scala, you most likely do not ne
 * Yield operator
 * Actors
     * Kotlin supports [Quasar](http://www.paralleluniverse.co/quasar/), a third-party framework for actor support on the JVM
-* Parallel collections
-    * Kotlin supports Java 8 streams, which provide similar functionality
+* Collections
+    * Scala has its own unique collections library, while Koltin relies on Java collections with some extensions
```

Comment:
I think it should be formulated like "Scala has its own comprehensive hierarchy of collection types, incompatible with java.util.* collections, while Kotlin just utilizes java.util.* collections with some enhancements and extensions" (possible link to Collections page).

I'm sure there's a ton of differences could be derived from that fact, why do you think the particular two of them (namely about lists and traversable) deserve emphasis?